src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -897,6 +898,17 @@ public void init(final boolean forceNewConnection) {
             }
         }
 
+        if (appConfig.getInstanceTelemetry()) {
+            // If we have at least one check add jmxfetch_telemetry check
+            if (newInstances.size() >= 1) {
```

Comment:
Lets fold this into the above `if` statement with a `&&` to save some indentation levels.